File: app/src/main/java/fudi/freddy/mokitosample/store/StoreContract.java
package fudi.freddy.mokitosample.store;

/**
 * @version 1.0
 * @autor Romell Domínguez
 * @date 8/13/17
 */

public interface StoreContract<T> {
    void save(T object);

    T load(String mail);

}

File: app-kotlin/src/main/java/fudi/freddy/mokitosample/store/StoreContract.kt
package fudi.freddy.mokitosample.store

/**
 * @version 1.0
 * @autor Romell Domínguez
 * @date 7/25/20
 */
interface StoreContract<T> {
    fun save(`object`: T)
    fun load(mail: String): T
}